### DISCUSSION TITLE
chatwoot interactive message not displaying through API self hosted

### DISCUSSION THREAD

**Discussion by lexemekenya** [chatwoot/chatwoot#5439] [Chatwoot APIs]
Hi,

I am trying to create an interactive message via API on our self-hosted setup but no message gets into Chatwoot inbox.

**Comment by sojan-official** [comment]
@lexemekenya  could you share the payload that you are trying to send ? as well as screenshots / other errors if any

**Reply by lexemekenya** [reply]
Thanks @sojan-official ,

Please find below as requsted

![image](https://user-images.githubusercontent.com/28090322/190368736-6cf041c6-b94a-405e-a7e5-23144fbb2f16.png)

![image](https://user-images.githubusercontent.com/28090322/190368999-df2d9fbc-dc3b-4504-b6f3-bfffe3161fd5.png)

The response is as below

{
    "id": <REDACTED>,
    "content": "card message",
    "inbox_id": <REDACTED>,
    "conversation_id": 14,
    "message_type": 0,
    "content_type": "cards",
    "content_attributes": {
        "items": [
            {
                "media_url": "https://assets.ajio.com/medias/sys_master/root/hdb/h9a/13582024212510/-1117Wx1400H-460345219-white-MODEL.jpg",
                "title": "Nike Shoes 2.0",
                "description": "Running with Nike Shoe 2.0",
                "actions": [
                    {
                        "type": "link",
                        "text": "View More",
                        "uri": "google.com"
                    },
                    {
                        "type": "postback",
                        "text": "Add to cart",
                        "payload": "ITEM_SELECTED"
                    }
                ]
            }
        ]
    },
    "created_at": 1663234260,
    "private": false,
    "source_id": null,
    "sender": {
        "additional_attributes": {},
        "custom_attributes": {
            "Account_type": "Business"
        },
        "email": null,
        "id": <REDACTED>,
        "identifier": null,
        "name": "<REDACTED>",
        "phone_number": "<REDACTED>",
        "thumbnail": "",
        "type": "contact"
    }
}

**Reply by lexemekenya** [reply]
What I see in Chatwoot

![image](https://user-images.githubusercontent.com/28090322/190369776-a868a45d-393e-4a3b-81f3-0a6d0765cbdd.png)

**Reply by sojan-official** [reply]
@lexemekenya  interactive messages are only rendered on client side widget at the moment. the agent dashboard display a very minimal version.  try sending message_type: "outgoing" instead of the "incoming"

**Reply by lexemekenya** [reply]
Hi @sojan-official, thanks for your support, I think for now I will just wait until in the future when you enable the agent dashboard to display it for my usecase is based on that. I wanted to send out a bulk message campaign to customers via API and I wanted the agents to see what message (looking nice as is the case with interactive messages) was sent out that triggered the feedback from the customer

**Comment by AdsonCicilioti** [comment]
@sojan-official any progress with this feature?
 I also have this need.  Cards on the dashboard side.
 Grateful!
